gems/aws-sdk-sqs/lib/aws-sdk-sqs/queue_poller.rb

```diff
@@ -406,7 +406,7 @@ def delete_messages(messages)
 
       def get_messages(config, stats)
         config.before_request&.call(stats)
-        messages = send_request(config).messages
+        messages = send_request(config).messages || []
```

Comment:
I understand this as a temp fix, but shouldn't messages be parsed as an empty array already?

Reply:
I don't think so - I think in aws json nil array members should be desrialized as nil and not default to `[]`.

Reply:
Then this is a breaking change to the service - `receive_message` API has a different return value, and breaks other usages, like sidekiq.

Reply:
I added a plugin as a temporary solution? But I think when we see query compatible trait and also a list, we need to default it to an empty list.

Reply:
I see that this change was backed out, but nevertheless wish to note for future reference that, if one's style guide / cyborg law enforcement officers permit, the `Kernel#Array` method is a nice way to guarantee that one has an Array :)